src/main/resources/citizen/templates/document/defendantResponseReceipt.html

```diff
@@ -429,13 +429,15 @@
           {{ repaymentPlan.paymentSchedule }}
         </td>
       </tr>
+      {% endif %}
+
       <tr class="no-border">
-        <td><strong>Make the first payment on:</strong></td>
+        <td><strong>Final payment by:</strong></td>
         <td>
-          {{ repaymentPlan.firstPaymentDate }}
+          {{ whenWillTheyPay }}
```

Comment:
Again should this be whenWillTheyFinishPaying

Reply:
Fixed